### ISSUE TITLE
Bug: hard-coded paths in make_conf_*.py files cause crash

### ISSUE THREAD

**Issue by saraghsm** [saraghsm/sound-of-failure#9]
The hard-coded paths are in [src/00_utils/make_conf_base.py](https://github.com/saraghsm/sound-of-failure/blob/7ca7dded4a586ad9ef6b2c922580e8c84af00ac7/src/00_utils/make_conf_base.py) in [#L22](https://github.com/saraghsm/sound-of-failure/blob/7ca7dded4a586ad9ef6b2c922580e8c84af00ac7/src/00_utils/make_conf_base.py#L22) and [#L26](https://github.com/saraghsm/sound-of-failure/blob/7ca7dded4a586ad9ef6b2c922580e8c84af00ac7/src/00_utils/make_conf_base.py#L26) and in [/src/00_utils/make_conf_convAE.py](https://github.com/saraghsm/sound-of-failure/blob/7ca7dded4a586ad9ef6b2c922580e8c84af00ac7/src/00_utils/make_conf_convAE.py) in [#L65](https://github.com/saraghsm/sound-of-failure/blob/7ca7dded4a586ad9ef6b2c922580e8c84af00ac7/src/00_utils/make_conf_convAE.py#L65) and [#L67](https://github.com/saraghsm/sound-of-failure/blob/7ca7dded4a586ad9ef6b2c922580e8c84af00ac7/src/00_utils/make_conf_convAE.py#L67). 

The problem is that while everything inside the base directory of the project (`sound-of-failure/`) has of course a predefined structure, everyone might structure their own Google drive space differently. But these lines have assumed a specific structure inside the `/gdrive/MyDrive/` directory. For example, my path is: `/gdrive/MyDrive/DSR_SOF/sound-of-failure/...`

Perhaps this is what we need:
```
google_drive_dir = /gdrive/MyDrive/    ---> hard coded
project = sound-of-failure/   ---> hard coded
raw_data_dir = google_drive_dir\<path to raw data directory inside the Google Drive that is asked from user via the cmd line>
base_dir = google_drive_dir\<path to project directory inside the Google Drive that is asked from user via the cmd line>\project
```

(For now I've changed the paths manually inside the scripts for myself.)

**Comment by saraghsm** [comment]
oh, or may be we don't event need the `google_drive_dir `and `project` parameters...

**Comment by wrijupan** [comment]
Thanks for pointing it out. We can make it generic. How about doing away with any dependency on gdrive? So if someone wants to run things on a local machine that should still work. The cmd-line inputs should be:

 -raw : the location of the raw data dir
 -prj : the directory hosting sound-of-failure (i.e. the path up to but excluding sound-of-silence)

You can view the changes [here](https://github.com/saraghsm/sound-of-failure/commit/05b9cf28f2402697d635f8da59f1a2fdffaf77f8).

An example of usage will be -

`%run /gdrive/MyDrive/sound-of-failure/src/00_utils/make_conf_base.py -raw '/gdrive/MyDrive/Hitachi_data/dataset' -prj '/gdrive/MyDrive'
`

`%run /gdrive/MyDrive/sound-of-failure/src/00_utils/make_conf_convAE.py -prj '/gdrive/MyDrive' -ae VAE -mel 32 -fft 1024 -hop 256 -dim 64 -s 8`

Let me know if you agree.

**Comment by saraghsm** [comment]
Yes, it's a better solution! It's specially good that we don't make any assumption of Google Drive or local machine etc.

**Comment by NikoHobel** [comment]
Agree with that suggestion.

**Comment by wrijupan** [comment]
Okay! I have pushed the updated notebooks ([AE](https://github.com/saraghsm/sound-of-failure/blob/main/notebooks/WB_20210325_ConvAE_training_demo.ipynb) and [VAE](https://github.com/saraghsm/sound-of-failure/blob/main/notebooks/WB_20210325_VarAE_training_demo.ipynb)) compatible with the modified config files. 

I will close the issue if you have no further comments regarding the path.

**Comment by saraghsm** [comment]
Thanks! Just one tiny little detail: explanation of the command line parameters should be changed accordingly if needed (for example maybe [hera](https://github.com/saraghsm/sound-of-failure/blob/1567c9334036c0cc7fd15102472af96419965e03/src/00_utils/make_conf_base.py#L14) )
